man/adoc/bpftrace.adoc

```diff
@@ -1720,6 +1720,16 @@ kprobe:do_nanosleep
  */
 ----
 
+[#functions-len]
```

Comment:
this also should get added to the table:
https://github.com/bpftrace/bpftrace/blob/master/man/adoc/bpftrace.adoc#functions